deploy/platform/localhost.go

```diff
@@ -190,6 +207,9 @@ func (d *Localhost) Stop() error {
 	if err != nil {
 		dbg.Lvl3("Error stopping localhost", err)
 	}
+
+	// Wait for eventual connections to clean up
+	time.Sleep(time.Second)
 	return nil
 }
```

Comment:
There is no need for both Wait() + Stop() since basically they do the same thing, one is blocking whereas the other is not.
 We could somehow use a Wait() for regular "stops" of the platform and a Clean / Cleanup () for killing everything the hardway (the way Stop() is done right now), when a process can not finish after a certain timeout.
Another possiblity is to transform Wait() to take a timeout. If the timeout is passed, then Wait() kills the processes and return back to the caller. That way, both functionalities are present and there is no one more method to the interface.